src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -835,14 +835,68 @@ static bool processAreaScope(const Scope* scoP, BSONObj &areaQuery)
 }
 
 
+
+
+/* *****************************************************************************
```

Comment:
Not an actual change in addFilterScope() and sortCriteria(). Actually, I think that I dindn't move them, what I do was to remove `static` in processAreaScopeV2()  and move that function (but not the others).

However, the diff is shown in this weird way...